src/Components/Components/src/Microsoft.AspNetCore.Components.csproj

```diff
@@ -22,6 +22,10 @@
     <Reference Include="Microsoft.Extensions.DependencyInjection.Abstractions" />
   </ItemGroup>
 
+  <ItemGroup>
+    <InternalsVisibleTo Include="Microsoft.AspNetCore.Components.Web" />
+  </ItemGroup>
```

Comment:
This comes out of some discussions we had within the team recently. Benefits:

 * Will simplify the layering in Mackinnon's head components work (original reason)
 * Here, it helps a lot with sharing some of the stuff about discovering component parameters and building `ParameterView` instances efficiently into `.Web` without unwanted extra public API
 * Here, it also helps avoid whole extra "handle"-type patterns for letting a renderer give out access to certain facilities like updating root component parameters without forcing *all* renderers to expose that unconditionally

We discussed that we don't intend to increase the use of IVTs broadly, but the Components/Components.Web pairing is particularly special.